Scroller/EditableWrapper.js

```diff
@@ -213,14 +239,18 @@ const EditableWrapper = (props) => {
 
 	const handleKeyDown = useCallback((ev) => {
 		const {keyCode, target} = ev;
-		const {selectedItem} = mutableRef.current;
-		if (is('enter', keyCode)) {
+		const {selectedItem, spotlightId} = mutableRef.current;
+		const targetItemNode = findItemNode(target);
```

Comment:
`findItemNode()` returns `undefined` if apps don't set `dataset.index` properly, then `targetItemNode` will have an invalid value. It will make a run-time error inside `startingEditing()` since the function does not check `item` exists or not.
What about adding some guard for a wrong usage?

Reply:
Thanks. I fixed.